tests/testthat/test-epidist.R

```diff
@@ -1137,3 +1137,36 @@ test_that("as.function fails as expected", {
     regexp = "Cannot convert unparameterised <epidist> to distribution function"
   )
 })
+
+test_that("as.data.frame works for <epidist>", {
+  # message about missing citation suppressed
+  edist <- suppressMessages(epidist(
+    disease = "ebola",
+    epi_dist = "incubation",
+    prob_distribution = "gamma",
+    prob_distribution_params = c(shape = 1, scale = 1)
+  ))
+  df <- as.data.frame(edist)
+  expect_s3_class(df, class = "data.frame")
+  expect_identical(dim(df), c(1L, 10L))
```

Comment:
I would suggest defining the expected numbers of rows and columns as a variable and passing it here, as where these come from might be confusing for future maintainers - you could reuse the vector of expected column names below.

Reply:
I've just implemented this idea using variables to hold the number of columns and rows expected, however it feels a bit over the top. I think that the expectation is still readable as a comparison of the dimensions to the vector. Will leave as is for now.